### PR TITLE
DEVELOPER-810 - DO NOT MERGE - Updated Blinkr.yaml to ignore broken links that have JIRA tickets open.

### PR DESCRIPTION
**Pull request by Dantheman720**
Testing all ignored links.

### CODE REVIEW

**Comment by Dantheman720** [comment]
retest this please

**Comment by paulrobinson** [comment]
What's the status of this?

**Comment by Dantheman720** [comment]
I believe this was about up to date.  I will run through that now and finalize the PR for merge.

File: _config/blinkr.yaml [hunk 1]
```diff
@@ -137,33 +82,180 @@ ignores:
...
+  - url: https://devstudio.jboss.com/central/install?connectors=org.jboss.tools.aerogear.hybrid #JIRA Ticket Outstanding
+#  - url: https://access.redhat.com/site/documentation/en-US/JBoss_Portal_Platform/6.1/html/Development_Guide/chap-Portlet_Bridge.html #JIRA Ticket Outstanding
+  - url: http://www.jboss.org/jdf/quickstarts/jboss-as-quickstart/guide/Introduction/ #JIRA Ticket Outstanding
+#  - url: https://access.redhat.com/site/documentation/en-US/JBoss_Portal_Platform/6.1/html/Development_Guide/Twitter.html #JIRA Ticket Outstanding
+  - url: http://www.jboss.org/feeds/view/jbossprofiler #JIRA Ticket Outstanding
+  - url: https://github.com/jboss-developer/jboss-eap-quickstarts/fork_select #JIRA Ticket Outstanding
+  - url: https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Fuse/6.1/html/Apache_Camel_Component_Reference/files/_IDU_Jing.html #JIRA Ticket Outstanding
+#  - url: http://community.jboss.org/en/jbossas/picketbox?view=discussions #JIRA Issue SECURITY-852 (Picketbox) - User Forum, Community ----- Still Outstanding
```
Comment (paulrobinson) [inline _config/blinkr.yaml]: What's your reasoning for commenting our, rather than deleting?

**Pull request merged**